lib/honeybadger/notice.rb

```diff
@@ -64,21 +74,31 @@ class Notice
 
     # The exception cause if available.
     attr_reader :cause
```

Comment:
All of these methods should be marked `# @api public` so that our YARD doc generates documentation for them.